maui-toc.html

```diff
@@ -1262,6 +1262,25 @@
             <li><a href="/maui/Slider/events-and-commands">Events and Commands</a></li>
         </ul>
     </li>
+    <li>
+        Smart Text Editor
+        <ul>
+            <li><a href="/maui/SmartTextEditor/overview">Overview</a></li>
+            <li><a href="/maui/SmartTextEditor/getting-started">Getting Started</a></li>
+            <li><a href="/maui/SmartTextEditor/suggestion-display-mode">Suggestion Mode</a></li>
+            <li><a href="/maui/SmartTextEditor/customization">Customization</a></li>
```

Comment:
Overview
Getting started
Suggestion Mode
Events And Commands
Custom AI Service
Customization


add this order